admin/scripts/sync_projects/README.md

```diff
@@ -0,0 +1,54 @@
+# Description
+
+Scripts to automatically open PRs for projects and track them in issues.
+
+# Setup
+
+## Credentials
+
+Create a [fine-grained token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#creating-a-fine-grained-personal-access-token) with the following permissions:
+
+- Contents: Read & write
+- Issues: Read & write
+- Pull Requests: Read & write
+
+Put the token in a file with the name `GH_TOKEN` in your crendentials directory. Example:
+
+```sh
+$ cat .env/GH_TOKEN
+<TOKEN>
+```
+
+## NLnet Dashboard
+
+Activate the development environment:
+
+```sh
+nix-shell
+```
+
+Run the extraction script on a directory containing the JSON files exported from the NLnet dashboard:
+
+```sh
+extract-project-data ./directory >>projects.json
+```
+
+## Notion
+
+Get the project list from Notion:
+
+- Navigate to the [Projects list](https://www.notion.so/nixos-foundation/15759d49e1be808186e5dc8c2c600ba8?v=9e8141539d9c41ad98ab2368b12d030f) page
+- If you can't access the page, you should request permissions
+- In the database table, make sure that the `Subgrants` field is in the view
+- Click on the top-right menu and export as a csv, choosing the default options
+- Save the ZIP file to your filesystem
+
+# Usage
+
+With the development environment active, run:
+
+```sh
+sync-projects --help
+```
+
+and follow the instructions.
```

Comment:
Add Development section containing instructions how to run tests and other relevant info.

Reply:
I'll add this once the tests are set up properly. This should probably be in a separate `CONTRIBUTING.md` file.